HIRS_Utils/src/main/java/hirs/validation/SupplyChainCredentialValidator.java

```diff
@@ -285,7 +285,7 @@ public AppraisalStatus validateDeltaPlatformCredentialAttributes(
             return new AppraisalStatus(FAIL, message);
         }
         if (basePlatformCredential == null) {
-            message = baseErrorMessage + "an endorsement credential";
+            message = baseErrorMessage + "an base platform credential";
```

Comment:
Just nit pick'in..."a base platform" instead of "an base platform"?

Reply:
Updated.